Slim/Utils/Prefs.pm

```diff
@@ -96,6 +96,7 @@ my $os = Slim::Utils::OSDetect->getOS();
 $os->migratePrefsFolder($path);
 
 my $prefs = preferences('server');
+my $ebmPrefs = preferences('plugin.extendedbrowsemodes');
```

Comment:
Again: no plugin specific code here. The change handler can be registered in the plugin itself.